src/types/data/heartBeat.ts

```diff
@@ -0,0 +1,8 @@
+import type { Name } from '../../routes/profile/validation.ts';
+import type { Access } from './access.ts';
+
+export interface Heartbeat {
+	name: Name;
```

Comment:
```suggestion
	name: Profile['name'];
```

Note: It is either bracket, or dot notation for referencing a type's property.